.github/workflows/lighthouse.yml

```diff
@@ -11,52 +11,7 @@ concurrency:
 
 jobs:
   build-plugin-lh:
-    name: Build plugin LH
-    runs-on: ubuntu-latest
-    if: startsWith( github.repository, 'elementor/' )
-    outputs:
-      changelog_diff: ${{ steps.changelog_diff_files.outputs.diff }}
-    steps:
-      - name: Checkout source code
-        uses: actions/checkout@v4
-      - name: Check if this is only a changelog PR
-        id: changelog_diff_files
-        uses: technote-space/get-diff-action@v6
-        with:
-          # PATTERNS are:
-          # Everything: **/*
-          # Everything in directories starting with a period: .*/**/*
-          # Not readme.txt: !readme.txt
-          # Not changelog.txt: !changelog.txt
-          PATTERNS: |
-            **/*
-            .*/**/*
-            !readme.txt
-            !changelog.txt
-      - name: Install Node.js 20.x
-        uses: actions/setup-node@v4
-        with:
-          node-version: 20.x
-      - name: Cache node modules
-        if: github.event.pull_request.title == null || steps.changelog_diff_files.outputs.diff
-        uses: actions/cache@v4
-        with:
-          path: ~/.npm
-          key: npm-${{ runner.os }}-${{ hashFiles('**/package-lock.json') }}
-      - name: Install dependencies
-        if: github.event.pull_request.title == null || steps.changelog_diff_files.outputs.diff
-        run: npm ci
-      - name: Build
-        if: github.event.pull_request.title == null || steps.changelog_diff_files.outputs.diff
-        run: npx grunt build
-      - name: Save build to cache
-        if: github.event.pull_request.title == null || steps.changelog_diff_files.outputs.diff
-        uses: actions/cache@v4
-        id: restore-build
-        with:
-          path: ./build/*
-          key: ${{ github.sha }}
-
+    uses: ./.github/workflows/build.yml
```

Comment:
Instead of this, can we cache/keep the results of the build job and reuse them?

Reply:
@yotamselementor do you have any suggestions? 
What I've investigated: 
1. We have an option to trigger `workflow_run` event after the build job is done, but this works only on default branch, so we will run playwright not on PR's branch, but on main. 
2. Trigger workflow run via API. This works fine, but then playwright workflow will be detached from a PR, so if it fails, user can still merge and we would need to add a trigger for any new workflow to build job.